src/main/java/seedu/address/logic/commands/AnswerCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONFIDENCE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.card.exceptions.NoCardSelectedException;
+
+/**
+ * Adds a tag to the address book.
+ */
+public class AnswerCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "answer";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Answer a selected flashcard. "
+            + "Parameters: "
+            + PREFIX_CONFIDENCE + "CONFIDENCE LEVEL";
+
+    public static final String MESSAGE_SUCCESS = "Your card has been scheduled.";
+    public static final String MESSAGE_CARD_NOT_SELECTED = "Cannot answer to no card, please select a card first.";
+
+    private final int confidenceLevel;
+
+    /**
+     * Creates an AddCommand to add the specified {@code Card}
+     */
+    public AnswerCommand(int confidenceLevel) {
+        this.confidenceLevel = confidenceLevel;
+    }
+
+    @Override
+    public CommandResult executeUndoableCommand() throws CommandException {
+        requireNonNull(model);
+        //requireNonNull(model.getSelectedCard());
```

Comment:
is this a temporary or permanent removal btw?

Reply:
permanant removal